compiler/src/iree/compiler/Codegen/Dialect/VectorExt/Transforms/VectorExtFoldUnitExtentDims.cpp

```diff
@@ -62,8 +62,7 @@ struct DropToLayoutUnitDims final
     auto newToLayoutOp = rewriter.create<IREE::VectorExt::ToLayoutOp>(
         loc, rankReducedValue.getType(), rankReducedValue, newLayout,
         toLayoutOp.getSharedMemoryConversion());
-    newToLayoutOp->setDiscardableAttrs(
-        toLayoutOp->getDiscardableAttrDictionary());
+    newToLayoutOp.setIntrinsicAttr(toLayoutOp.getIntrinsicAttr());
```

Comment:
This should be passed to the constructor instead.

Reply:
done!